### DISCUSSION TITLE
📣 Call for participants: Sharing Crossplane success stories!

### DISCUSSION THREAD

**Discussion by cwilhit** [crossplane/crossplane#3182] [General]
### Are you using Crossplane in production to power your control planes?
The community would benefit from users sharing their stories about how Crossplane is helping solve business problems. We should publish a series of blog posts to the [Crossplane blog](https://blog.crossplane.io/) to evangelize how crossplane has solved problems for users in the community. I think sharing these successes would help galvanize others to take the plunge into Crossplane.

The only criteria should be that users are:

- using crossplane 
- to support systems & services running in production. 

I will happily take point on this. If you are interested in sharing your success story to have published on the crossplane blog, please reach out to me in the crossplane slack community `@Craig D Wilhite` to collaborate on a case study (architecture, problem, resolution with crossplane).

**Comment by kferrone** [comment]
Oh I've done some things with Crossplane.

**Reply by cwilhit** [reply]
😀 in personal or professional capacity? If it's the latter, I think it could make a good candidate to go into a case study.

**Reply by kferrone** [reply]
Professionally in production at two different companies. I also got `experimental` with my home lab.
